html/publications.html

```diff
@@ -0,0 +1,222 @@
+                                learning and automatic evaluation of computer programming skills.". In EDULEARN
+                                2018.</small></p>
+                        </div>
+                    </li>
+                    <li class="list-group-item border-bottom bg-grey">
+                        <h2 class="">Continuous assessment in a computer programming course supported by a software
+                            tool</h2>
+                        <p class="">Este documento hace una introducción a UNCode, una plataforma para el
+                            aprendizaje de la programación de computadores mediante el uso de un evaluador
+                            automático o juez. En el artículo se menciona las principales motivaciones por las
+                            cuales es necesario la implementación de un juez orientado a la enseñanza en contraste a
+                            los jueces orientados a la competición. Además, se explica los beneficios y herramientas
+                            que tiene UNCode para los estudiantes y docentes. </p>
+                        <p>Mediante un análisis cuantitativo se estudia la interacción entre los estudiantes y la
+                            herramienta de evaluación. Particularmente se aprecia las soluciones propuestas por los
+                            estudiantes en relación a la respuesta que retorna el juez. También se analiza la tasa
+                            de éxito, la cual consiste en el porcentaje soluciones correctas en contraste al número
+                            de intentos del estudiante. Adicionalmente se toma en cuenta la percepción de los
+                            estudiantes hacia el uso en de una herramienta evaluadora y su aplicación en el proceso
+                            de aprendizaje mediante la aplicación de encuestas.</p>
+                        <div>
+                            <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/cae.22058" class="py-3">Ir al
+                                documento</a>
+                            <a onclick="cite('uncode-cite-2-container',this.id)" id="uncode-cite-2" type="button"
+                               role="button"
+                               class="link-style popovers px-3" data-toggle="tooltip" data-placement="right"
+                               title="Referencia copiada" aria-controls="#uncode-cite-2-container">Cómo Citar</a>
+                        </div>
+                        <div class="row px-4 collapse py-2" id="uncode-cite-2-container">
+                            <p class="font-italic box-line p-1" id="uncode-cite-2-cite"><small>Felipe Restrepo-Calle,
+                                Jhon J. Ramírez Echeverry, Fabio A. González. "Continuous assessment in a computer
+                                programming course supported by a software tool," in CAEE, 27(1), 80-89, 2019.</small>
```

Comment:
```suggestion
                            <p class="font-italic box-line p-1" id="uncode-cite-2-cite"><small>Restrepo‐Calle, F, Ramírez Echeverry, JJ, González, FA. Continuous assessment in a computer programming course supported by a software tool. Comput Appl Eng Educ. 2019; 27: 80– 89. https://doi.org/10.1002/cae.22058</small>
```
@ferestrepoca is this change correct?

Reply:
Yes, it is correct.